charts/karpenter/templates/clusterrole.yaml

```diff
@@ -40,3 +40,7 @@ rules:
     resources: ["mutatingwebhookconfigurations"]
     verbs: ["update"]
     resourceNames: ["defaulting.webhook.karpenter.k8s.aws"]
+  # Write
+  - apiGroups: ["karpenter.k8s.aws"]
+    resources: ["awsnodetemplates/status"]
+    verbs: ["patch", "update"]
```

Comment:
Do you need both? 

On another note, I've seen issues with updating status where the update can cause the controller to re-reconcile, which can have a runaway effect. Can you verify that this isn't happening?

Reply:
@engedaam If we need to avoid this, we should use a GenerationChangedPredicate event filter to filter out Update events that come from status updates.

Reply:
That was a problem that was occurring. It was fixed with a GenerationChangedPredicate event filter.